src/builder/modules/Generator.res

```diff
@@ -122,8 +122,49 @@ let usePage = id => {
   model := Builder.usePage(model.contents, id)
 }
 
-let fillBackgroundColor = (fillStyle: string) => {
-  model := Generator_Builder.fillBackgroundColor(model.contents, fillStyle)
+let fillBackgroundColor = (color: string) => {
+  model := Generator_Builder.fillBackgroundColor(model.contents, color)
+}
+
+let fillBackgroundColorWithWhite = () => {
+  model := Generator_Builder.fillBackgroundColor(model.contents, "#ffffff")
+}
+
+let drawLine = (
+  from: Builder.position,
+  to: Builder.position,
+  ~color: string="#000000",
+  ~width: float=1.0,
+  ~pattern: array<int>=[],
+  ~offset: int=0,
+  (),
+) => {
+  model := Generator_Builder.drawLine(model.contents, from, to, ~color, ~width, ~pattern, ~offset)
+}
+
+let drawFold = (from: Builder.position, to: Builder.position) => {
+  model :=
+    Generator_Builder.drawLine(
+      model.contents,
+      from,
+      to,
+      ~color="#7b7b7b",
```

Comment:
Any reason to choose this particular color?

Reply:
I'm not entirely sure, but I think that it was first used by Rooterbuster in the ultimate bendable gen(albeit with some transparency), and since then most folds have been done using that color.